mod_wtjshoppingfavorites/src/Helper/WtjshoppingfavoritesHelper.php

```diff
@@ -0,0 +1,62 @@
+<?php
+/**
+ * WT JoomShopping Favorites is an alternative wish list (favorite products) for JoomShopping based on cookies.
+ * @package     WT JoomShopping Favorite
+ * @version     2.0.2
+ * @Author      Sergey Tolkachyov, https://web-tolk.ru
+ * @copyright   Copyright (C) 2024 Sergey Tolkachyov
+ * @license     GNU/GPL 3.0
+ * @since       1.0.0
+ * @link        https://web-tolk.ru/en/dev/joomshopping/wt-joomshopping-favorite.html
+ */
+namespace Joomla\Module\Wtjshoppingfavorites\Site\Helper;
+
+\defined('_JEXEC') or die;
+
+use Joomla\CMS\Factory;
+use Joomla\Registry\Registry;
+
+class WtjshoppingfavoritesHelper
+{
+	/**
+	 * Method to get last seen products
+	 *
+	 * @param   Registry  &$params  Module parameters
+	 *
+	 * @return  array  Last seen products
+     * @since 1.0.0
+	 */
+	public static function getProductList(&$params, $app): array
+	{
+        $cookie = $app->input->cookie->get('wtjshoppingfavorites', null, $filter = 'string');
+		if (!empty($cookie))
+		{
+			$product_ids = unserialize($app->input->cookie->get('wtjshoppingfavorites', null, $filter = 'string'));	
```

Comment:
В Joomla 4 и Joomla 5 input получать уже через метод нужно. просто `$app->input` уже `deprecated`
```suggestion
			$product_ids = unserialize($app->getInput()->cookie->get('wtjshoppingfavorites', null, $filter = 'string'));	
```